### DISCUSSION TITLE
That's this: https://github.com/directus/next/issues/936

### DISCUSSION THREAD

**Discussion by kasparpalgi** [directus/directus#8648] [Feature Requests]
Both URLs not working:

That's this: https://github.com/directus/next/issues/936

And here is how it was done in v8 in case you want to try a PR:
https://github.com/directus/app/blob/master/src/interfaces/preview/input.vue

_Originally posted by @benhaynes in https://github.com/directus/directus/discussions/2987#discussioncomment-129170_

**Comment by rijkvanzanten** [comment]
directus/next became directus/directus, directus/app became directus/v8-archive 👍🏻 

Like Ben mentioned in that follow up comment, this already exists currently:

> We have this now, it's the "Button Links" Interface.

so there's no more need for any PRs or implementation requirements 👍🏻
